docs/README.md

```diff
@@ -1,49 +1,55 @@
+### Policy Management:
 
-- [usage](reference/usage.md) - Additional information for using the `ratify` executable
-- [gatekeeper-policy-authoring](reference/gatekeeper-policy-authoring.md) - Authoring gatekeeper policies for use with ratify, including rego references/examples.
-- [oras-auth-provider](reference/oras-auth-provider.md) - Explanation of various authentication mechanisms available for use with ratify.
-- [creating-plugins](reference/creating-plugins.md) - Details on creating your own plugins for use with ratify.
-- [CRD references](reference/crds/) - Describes the required and optional properties of ratify CRDs
-- [instrumentation](reference/instrumentation.md) - Details on the current supported instruments and metric provider setup guides
-## developer
+- [gatekeeper-policy-authoring](quickstarts/gatekeeper-policy-authoring.md) - Authoring gatekeeper policies for use with ratify, including rego references/examples.
+
+### SPDX Integration:
+
+- [working-with-spdx](quickstarts/working-with-spdx.md) - Use ratify with syft, oras, and verification using ratify cli.
+
+### Contributing:
+- [Developer Getting Started](../CONTRIBUTING.md) - Getting started with ratify development.
 
-The documents in this directory are for developers who want to contribute to Ratify or want to understand the internals of Ratify.
+## Reference 
 
-- [Contributing](../CONTRIBUTING.md) - How to get ratify development environment setup and generally contribute back to Ratify.
-- [README](./developer/README.md) - breaks down the architecture of Ratify and how it works.
-- [providers](./developer/providers.md) - information about built-in providers and the extensible policy provider interface
-- [executor](./developer/executor.md) - information about the executor and how it works
-- [store](./developer/store.md) - information about the store plugin and how it works
-- [verifier](./developer/verifier.md) - information about the verifier plugin and how it works.
+Documentation for understanding ratify and its components:
 
-### docs improvements backlog
+Framework and configuration:
+- [Framework](reference/ratify-framework-overview.md) - The ratify framework is the core of ratify.  It is responsible for orchestrating the execution of the various plugins and providing a common interface for them to interact with each other.
+- [Configuration](reference/ratify-configuration.md) - Ratify's configuration consist of:
+  - [store](reference/store.md) - Ratify's store is responsible for storing and retrieving artifacts.
+  - [verifier](reference/verifier.md) - Ratify's verifier is responsible for verifying the integrity of artifacts.
+  - [provider](reference/providers.md) - Ratify's policy provider is responsible for providing policies to the framework.  Policies are used to determine if an artifact is valid or not.  Policies are written in [rego](https://www.openpolicyagent.org/docs/latest/policy-language/).
```

Comment:
```suggestion
  - [provider](reference/providers.md) - Ratify's policy provider is used by the framework to make a final decision on if an artifact is valid or not. Policies are defined via [configuration](reference/providers.md#policy-providers).
```

Reply:
The built in policy evaluation for Ratify is only config based for now. @binbin-li is working on lighting up Rego Support. Currently, GK is the component doing the policy evaluation and not Ratify. Ratify only provides a list of verification reports which are then processed by GK and rego is applied on top of those reports.

Reply:
@akashsinghal I feel the current implementation is a bit confusing to users. We have 2 actual policies right now, the rego policy on GK side, and config policy on Ratify side. Ratify's PolicyEnforcer will evaluate result first and return reports and overall result(IsSuccess) to GK. Then GK will make another evaluation on the reports and result based on the rego policy defined in the constraint policy.